cost/google/idle_vm_recommendations/google_vm_recommendations.pt

```diff
@@ -400,65 +397,81 @@ script "js_instance_cost_mapping", type:"javascript" do
+        var resource_link = vm.selfLink.replace("https://www.googleapis.com/compute/v1/", "")
+        vm["resourceLink"] = resource_link
+    })
+
+    _.each(recommendations, function(recommendation){
+      count ++
+      var rec_resource_link = recommendation.resourceLink.replace("//compute.googleapis.com/", "")
+      var vm = _.find(vms, function(v) {
+        return v.resourceLink == rec_resource_link
+      })
+      if (!(vm == undefined || vm == null)) {
+        if(recommendation['currency'] !== undefined ) {
+          if (recommendation['costUnits'] == null) {
+            recommendation['costUnits'] = 0
           }
-          if (recommendation.costNanos<0) {
-            recommendation.costNanos = recommendation.costNanos * -1
+          if (recommendation['costNanos'] == null) {
+            recommendation['costNanos'] = 0
           }
-          if (recommendation.costUnits<0) {
-            recommendation.costUnits = recommendation.costUnits * -1
+          if (ds_currency_reference[recommendation['currency']] !== undefined ) {
+          cur = ds_currency_reference[recommendation['currency']]['symbol']
+          } else {
+            cur = "$"
           }
-          strNano = "."+recommendation.costNanos
-          combine = recommendation.costUnits + parseFloat(strNano)
-          monthsDuration = parseFloat(recommendation.duration.replace("s",""))/2628288
-          monthlySavings = combine / monthsDuration
-          total = total+monthlySavings
-          vm['savings']= (Math.round( monthlySavings * 1000) / 1000)
           vm['savingsCurrency'] = cur
-          vm['priority'] = recommendation['priority']
-          vm['region'] = recommendations['region']
-          vm['primaryImpact'] = recommendation['primaryImpact']
-          vm['recommenderSubtype'] = recommendation['recommenderSubtype']
-          vm['state'] = recommendation['state']
-          vm['service'] = "Compute"
-          tags = []
-          if (vm['tags'] != null) {
-            if( typeof vm['tags'] == 'object') {
-              Object.keys(vm.tags).forEach(function(key) {
-                if (vm.tags[key] == null || vm.tags[key]=="") {
-                  vm.tags[key] = "null"
-                }
-                tags.push(String(key)+'='+String(vm.tags[key]))
-              });
-            } else {
-              tags.push(vm["tags"].replace(":","=null"))
-            }
+        } else {
+          cur = "$"
+          var separator = ","
+        }
+
+        //Convert Costs from Negative to Positive and add Units and Nanos
+        if (!(recommendation.costNanos == null || recommendation.costNanos == undefined)) {
+          if (recommendation.costNanos != 0) {
+            recommendation.costNanos = recommendation.costNanos * -1
           }
-          vm['tags'] = tags
-          instances.push(vm)
+        } else {
+          recommendation.costNanos = 0
         }
-      })
+
+        if (recommendation.costUnits<0) {
```

Comment:
```suggestion
        if (recommendation.costUnits != 0) {
```

Reply:
@nia-vf1 sorry I did not notice this change, I just applied this one.